test/Microsoft.AspNet.Mvc.Core.Test/Routing/AttributeRoutePrecedenceTests.cs

```diff
@@ -16,11 +17,25 @@ public class AttributeRoutePrecedenceTests
         [InlineData("Employees/{id}", "Employees/{employeeId}")]
         [InlineData("abc", "def")]
         [InlineData("{x:alpha}", "{x:int}")]
-        public void Compute_IsEqual(string xTemplate, string yTemplate)
+        public void ComputeMatched_IsEqual(string xTemplate, string yTemplate)
```

Comment:
Actually, the name of the underlying function we're testing is ComputeMatched, it's the local helper function that's named wrong.